tox.ini

```diff
@@ -1,27 +1,46 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3,docs
+    py27,
+    py33,
+    py34,
+    py35,
+    py36,
+    pypy,
+    pypy3,
+    docs,
+    coverage-report,
 
 [testenv]
+usedevelop = true
```

Comment:
Why is this necessary?

Usually we had a problem with namespace packages, when we used python setup.py test on Python >= 3.5, that would go away with usedevelop = true *or* if we switched to zope-testrunner.  Here I see you do both?